app/src/ui/release-notes/release-notes-dialog.tsx

```diff
@@ -8,11 +8,38 @@ import { ReleaseNote, ReleaseSummary } from '../../models/release-notes'
 import { updateStore } from '../lib/update-store'
 import { ButtonGroup } from '../lib/button-group'
 import { Button } from '../lib/button'
-import { LinkButton } from '../lib/link-button'
 
 import { Dialog, DialogContent, DialogFooter } from '../dialog'
 import { DialogHeader } from '../dialog/header'
 
+import { RichText } from '../lib/rich-text'
+import { Repository } from '../../models/repository'
+
+const repository = new Repository(
+  '',
+  -1,
+  {
+    dbID: null,
+    name: 'desktop',
+    owner: {
+      id: null,
+      login: 'desktop',
+      endpoint: 'https://api.github.com/',
+      hash: '',
+    },
+    private: false,
+    parent: null,
+    htmlURL: 'https://github.com/desktop/desktop',
+    defaultBranch: 'master',
+    cloneURL: 'https://github.com/desktop/desktop',
+    endpoint: 'https://api.github.com/',
```

Comment:
`endpoint` should be set to `https://api.github.com` since the `Tokenizer` is looking for that to return the right URL.

https://github.com/desktop/desktop/blob/50f7b88ff834119622018e953cd2f1fa444902c3/app/src/lib/text-token-parser.ts#L192

https://github.com/desktop/desktop/blob/50f7b88ff834119622018e953cd2f1fa444902c3/app/src/lib/api.ts#L808-L810

https://github.com/desktop/desktop/blob/50f7b88ff834119622018e953cd2f1fa444902c3/app/src/lib/api.ts#L827-L839

At the moment, clicking on an @-mention navigates you to api.github.com